internal/prompt/prompt.go

```diff
@@ -81,10 +101,24 @@ func interactiveInputError(message string) error {
 }
 
 // InputAndValidate prompts an input field and allows you to specfiy a custom validation function as well as the built in flags
-func (p *Prompt) InputAndValidate(title, message string, defaultResponse *string, validator ValidatorFunc, flags ...ValidatorFlag) (string, error) {
+// If the prompt is non-interactive, it returns defaultResponse.
+// If the prompt is forced, it returns forcedResponse if not nil, or defaultResponse.
+func (p *Prompt) InputAndValidate(title, message string, defaultResponse *string, forcedResponse *string, validator ValidatorFunc, flags ...ValidatorFlag) (string, error) {
+	if p.isForced {
+		response := forcedResponse
+		if response == nil {
+			response = defaultResponse
+		}
+		if response != nil {
+			p.out.Notice(locale.Tr("prompt_using_force", *response))
+			return *response, nil
+		}
+		return "", errs.New("No force option given for forced prompt")
```

Comment:
We should localize this through rationalized errors. We can do the rationalising here: https://github.com/ActiveState/cli/blob/8e47470c8c8b075aa0f23eca986631eec4f1a1f1/internal/captain/rationalize.go#L13-L13